### PR TITLE
Add Undo Command

### PR DESCRIPTION
**Pull request by tienyu2000**
Add undo command to revert work book back to previous state and undo command tests.

### CODE REVIEW

File: src/main/java/seedu/workbook/logic/parser/WorkBookParser.java [hunk 1]
```diff
@@ -6,15 +6,7 @@
...
-import seedu.workbook.logic.commands.AddCommand;
-import seedu.workbook.logic.commands.ClearCommand;
-import seedu.workbook.logic.commands.Command;
-import seedu.workbook.logic.commands.DeleteCommand;
-import seedu.workbook.logic.commands.EditCommand;
-import seedu.workbook.logic.commands.ExitCommand;
-import seedu.workbook.logic.commands.FindCommand;
-import seedu.workbook.logic.commands.HelpCommand;
-import seedu.workbook.logic.commands.ListCommand;
+import seedu.workbook.logic.commands.*;
```
Comment (rylzxc) [inline src/main/java/seedu/workbook/logic/parser/WorkBookParser.java:9]: Let's not use star imports as mentioned in the 2103T textbook

**Review by rylzxc — DISMISSED** [review]
Well done on implementing the `UndoCommand`!  I hope you got a better understanding on the codebase and how it works. 

Generally the code looks quite clean, but do remember to follow the checkstyle rules. Small matter but it reduces the amount of stuff we have to change in the future. :)

File: src/main/java/seedu/workbook/logic/parser/WorkBookParser.java (hunk 1 above)
Comment (tienyu2000) [reply src/main/java/seedu/workbook/logic/parser/WorkBookParser.java]: Okay, will fix this.

**Review by jacobkwan — APPROVED** [review]
LGTM, just the minor stuff about javadocs which can be fixed next time 
👍 👍

**Pull request merged**